mix.exs

```diff
@@ -53,7 +53,7 @@ defmodule Tower.MixProject do
 
       # Optional
       {:plug, "~> 1.0", optional: true},
-      {:bandit, "~> 1.6", optional: true},
+      {:bandit, github: "mtrudel/bandit", override: true, optional: true},
```

Comment:
this was just for testing edge.
Will wait for the next bandit published release

Reply:
Updated